src/shared/cloudformation/cloudformation.ts

```diff
@@ -775,8 +776,10 @@ const MANIFEST_URL = 'https://api.github.com/repos/awslabs/goformation/releases/
  * @param extensionContext
  */
 export async function refreshSchemas(extensionContext: vscode.ExtensionContext): Promise<void> {
-    CFN_SCHEMA_PATH = normalizeSeparator(path.join(extensionContext.globalStoragePath, 'cloudformation.schema.json'))
-    SAM_SCHEMA_PATH = normalizeSeparator(path.join(extensionContext.globalStoragePath, 'sam.schema.json'))
+    CFN_SCHEMA_PATH = normalizeSeparator(
+        path.join('file:/', extensionContext.globalStoragePath, 'cloudformation.schema.json')
```

Comment:
It's not really semantically correct to use path.join() to join a URI. And although the effect is intended, it's subtle and could be misunderstood as a mistake by a future reader: it will remove any `/` at the start of globalStoragePath, and more generally enforces a single `/`. 

I don't have a better suggestion, but we should at least include a brief 1-line comment above this line to mention that this is intentional.
```suggestion
        // path.join() is intentionally used here, we want a single "/" after "file:/". 
        path.join('file:/', extensionContext.globalStoragePath, 'cloudformation.schema.json')
```

Reply:
I'm not sure if `file:/` was working for Mac anyway. I'll probably just if statement this for non-Windows.